test/amqqueue_SUITE.erl

```diff
@@ -81,18 +116,21 @@ test_publish_unauthorized_error(Channel, _Client, Version) ->
         Err
```

Comment:
Yeah, the previous module checked the returned value of each testcase and expected `ok`. So the exception shouldn't be caught at all probably.